bubble-wrap.yaml

```diff
@@ -2692,7 +2695,6 @@ layers:
         # building footprints, pre-extrusion
         footprints:
             filter:
-                $zoom: { max: 18 }
```

Comment:
This was added way back when because if buildings are extruded we don't want to draw them all twice (which is a performance hit). Removing it here seems to allow buildings (footprints) to show up later when the extruded volumes aren't shown, but introduces a performance hit?